src/model.rs

```diff
@@ -79,8 +79,22 @@ pub struct DeactivateResponse {
     pub dbg_message: String,
 }
 
+#[derive(Clone, Copy, Deserialize, Debug, Eq, Hash, PartialEq, Serialize)]
+pub enum StatusColor {
+    #[serde(rename = "green")]
+    Green,
+    #[serde(rename = "orange")]
+    Orange,
+    #[serde(rename = "red")]
+    Red,
+    #[serde(rename = "grey")]
+    Grey,
```

Comment:
Quick clarification- does Serde add colors and you're renaming them? Little confused what this is doing with Serde

Reply:
This is like an enum variant name "override", so StatusColor::Green => "green" instead of the default "Green"